app/boards/native_posix.overlay

```diff
@@ -53,27 +54,26 @@
 
 		default: layer_0 {
 			label = "DEFAULT";
-			keys =
-			<
-				KC_A MT(MOD_LSFT, KC_B) KC_C KC_D
-				KC_E KC_F KC_G KC_H
-			>;
+
+			bindings = <
+				&kp B &mo 1 &kp C &kp E
+				&kp D &kp G &kp F &kp Q>;
 		};
 
 		lower: layer_1 {
 			label = "LOWER";
-			keys = <
-				KC_A KC_B KC_C KC_D
-				KC_E KC_F KC_G KC_H
-			>;
+
+			bindings = <
+				&cp M_NEXT &trans &kp O &kp Q
```

Comment:
We can come up with a better shortcut for this, but I now added `&cp` to reference the "Consumer Press" when someone pushes a consumer page key, which allows just using the actual consumer usage values directly, instead of the previous keycode hack. We could call this shorthand something else, e.g. `&mp` for "Media Press" since most end users think of these as media keys.